modules/book.js

```diff
@@ -0,0 +1,56 @@
+export default class Book {
```

Comment:
- [ ] [OPTIONAL] Your Code should be divided into reasonable modules, kindly divide this module more. you can create a module for adding books and one for removing a book